### ISSUE TITLE
Will this get an update to work with Tiled 1.2 Beta?

### ISSUE THREAD

**Issue by chaosdrop** [vnen/godot-tiled-importer#95]
**Plugin version**

2
**Issue description**

he JSON format has changed.
**Steps to reproduce**

**Sample map or tileset**

**Comment by vnen** [comment]
Eventually, yes, but only after a stable version is released. Also, I'm not really fond of maintaining two versions of the JSON loader, I'll probably update to the new format and ask people to update their Tiled editor (and that can only be done after the stable release).

**Comment by HeadClot** [comment]
The stable version is out. Just giving you a heads up.

https://www.mapeditor.org/2018/09/18/tiled-1-2-0-released.html

**Comment by zhengying** [comment]
does Tiled 1.2 supported?
When 2.3 version release?

**Comment by vnen** [comment]
TBH I don't know. I work on this plugin on my free time and I have almost no free time anymore. I will still review PRs if anyone is willing to do it.

**Comment by rosshadden** [comment]
@zhengying I have been using Tiled 1.2 since it's official release in September. 1.2.1, released in November, works as well. Are you asking because you found an issue or asking before you try?

As someone who has been programming for over 16 years I _very highly_ recommend just jumping into things and getting your feet wet. If it doesn't work, you can file an issue or even figure out what's wrong and fix it yourself; if it does work then all is well and you didn't need to post anything at all. :D

**Comment by zhengying** [comment]
@rosshadden  I tried yesterday, I found tmx file parse error, so I found here has an issue about tiled 1.2.

@rosshadden you said you have been using Tiled 1.2 , why your tmx file work fine?  did you fix something make it works ?

**Comment by zhengying** [comment]
Thanks @rosshadden !!  
You save me!

**Comment by vnen** [comment]
AFAIK only the JSON format has changed. So if you're using TMX it shouldn't be a problem. But the plugin still needs to be updated for the new JSON format.

**Comment by rosshadden** [comment]
I have only ever used the TMX format, so maybe something changed in the past few weeks that wasn't called out.

**Comment by zhengying** [comment]
> AFAIK only the JSON format has changed. So if you're using TMX it shouldn't be a problem. But the plugin still needs to be updated for the new JSON format.

but even use tmx format the tiles show not correct, I use jfmaror workaround method , works well now. 
can you merge this to master? thanks!
> Added "tilemap.format = 1" around line 222 of tiled_map_reader.gd after the TileMap.new()
link here:
[jfmajor 's a workround](https://github.com/godotengine/godot/issues/23631#issuecomment-442227273)

**Comment by vnen** [comment]
> but even use tmx format the tiles show not correct, I use jfmaror workaround method , works well now.

This is not related to the new Tiled format, but happens because of a recent change in Godot. This is already tracked in #103.

> can you merge this to master? thanks!

If someone make a PR, yes.

**Comment by zhengying** [comment]
thanks @vnen  I just see the #103 , 
I agree you that handle it after release 3.1 stable version.

**Comment by slevy85** [comment]
Hi,
what version of tiled map should we use then ?
